.github/actions/build_pandas/action.yml

```diff
@@ -3,15 +3,28 @@ description: Rebuilds the C extensions and installs pandas
 runs:
   using: composite
   steps:
-
     - name: Environment Detail
       run: |
-        conda info
-        conda list
+        if which mamba 2>/dev/null; then
+          mamba info
+          mamba list
+          mamba info | grep -Ei 'environment.+:' | grep -qEiv 'environment.+:.+none'
+        fi
+        python -VV
+        which python
+        which pip
+        pip list
```

Comment:
This seems overcomplicated. Not sure why the `if` condition is useful. And why the greps are useful after printing the whole `mamba info` (which includes the Python version already, which is printed later).

Are we missing any important information if we leave this as it was? Fine to use mamba instead of conda if that's what we use now, but unless something is really needed, better to keep things as simple as possible.

Reply:
We can remove the `if` (but will add back in a follow-up PR where we don't have Conda).

The greps are there to fail unless we have the correct version. Will add a comment.